src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2793,14 +2816,6 @@ namespace NServiceBus.Unicast
     [System.ObsoleteAttribute("Not a public API. Please use `MessageHandlerRegistry` instead. Will be removed in" +
         " version 7.0.0.", true)]
     public interface IMessageHandlerRegistry { }
-    public class MessageContext : NServiceBus.IMessageContext
```

Comment:
the only way to work with the context was by accessing the (now deprecated) MessageContext property on IBus which returns the IMessageContext interface. So I'm not sure why that class was even public in the first place. I think users can't really access that class so it should be safe to just make it private?